frontend-react/src/components/Admin/NewSetting.tsx

```diff
@@ -113,7 +98,7 @@ export function NewSetting({ match }: RouteComponentProps<Props>) {
                         form="edit-setting"
                         type="submit"
                         data-testid="submit"
-                        onClick={() => saveData()}
+                        onClick={async () => await saveData()}
```

Comment:
This is my code and I'm not sure the async/await is required here. CloudSonar hates it.